pyproject.toml

```diff
@@ -151,6 +151,9 @@ ignore = [
     # Pyflakes (F)
     "F403",  # ImportStarUsed
 
+    # flake8-fixme (FIX)
+    "FIX002",  # Line contains TODO | notes for improvements are OK iff the code works
```

Comment:
Why do we want to ignore this permanently?

Reply:
This means the CI doesn't fail if there's a TODO.
I think we agree that (in future) the CI should fail on FIXME, as broken things should be in production code,
but TODO and NOTE are more gray areas, up to the merging maintainer's discretion.

Reply:
e.g. ``TODO: speed up this code by refactoring in a certain way`` or ``TODO: refactor when py3.10+``, etc. IMO this is fine to merge.

Reply:
`FIX002` being ignored in `pyproject.toml` does not prevent anyone from addressing what the `TODO` comments suggest, so we can indeed keep it here, at least for now.